apps/parse/lib/parse/trip_updates.ex

```diff
@@ -54,6 +56,12 @@ defmodule Parse.TripUpdates do
     nil
   end
 
+  defp parse_uncertainty(%{uncertainty: uncertainty}) when is_integer(uncertainty) do
```

Comment:
Since your documentation (nice!) only allows three values, would it make sense to only allow those three values through? I think you can use a guard like `when is_integer(uncertainty) and uncertainty in [60, 120, 360]` but I haven't tested it.

Reply:
Does this incorporate the uncertainty values that Swiftly uses?